Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -268,6 +270,13 @@ private void UpdateViewState()
             {
                 ViewState = MasterDetailsViewState.Both;
             }
+
+            var after = ViewState;
```

Comment:
Instead of setting a variable and checking that variable, just call `ViewStateChanged?.Invoke(this, after);` within the if/else. Less code and less work